parser/prism/Translator.h

```diff
@@ -0,0 +1,55 @@
+#ifndef SORBET_PARSER_PRISM_TRANSLATOR_H
+#define SORBET_PARSER_PRISM_TRANSLATOR_H
+
+#include <memory>
+
+#include "../Node.h" // To clarify: these are Sorbet Parser nodes, not Prism ones.
```

Comment:
This would be the only `../` include in the codebase.

```suggestion
#include "parser/Node.h" // To clarify: these are Sorbet Parser nodes, not Prism ones.
```